backend/e4h-services/ingestion-service/app/utils/excel_utils.py

```
@@ -423,4 +428,95 @@ def add_non_blank_validations_to_file(
+                type="custom",
+                formula1=poc_formula,
+                allow_blank=True,
+                showErrorMessage=True,
+                errorTitle="Validation",
+                error=ERR_POC_USERNAME_REQUIRED_WHEN_ANGANWADI,
+            )
+            ws.add_data_validation(dv_poc)
+            dv_poc.add(f"{poc_letter}2:{poc_letter}{max_row}")
+
+        wb.save(file_path)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)